### PR TITLE
[JG] OPS-12383 - adding in clearing date to payment orchestrator model

### PR DESCRIPTION
**Pull request by DreamcastKnight**

### CODE REVIEW

File: app/controllers/des/DesController.scala [hunk 1]
```diff
@@ -48,6 +48,7 @@ class DesController @Inject() (
   def getFinancialData(vrn: Vrn): Action[AnyContent] = actions.securedAction(vrn).async { implicit request =>
     logger.debug("getFinancialData called")
     desConnector.getFinancialData(vrn).map { maybeFinancialDetails =>
+      println("frick" + maybeFinancialDetails.toString)
```
Comment (AndyHWChung) [inline app/controllers/des/DesController.scala:51]: probably don't need this anymore?

File: test/support/DesData.scala [hunk 2]
```diff
@@ -38,8 +38,10 @@ object DesData {
 
   val directDebitData: DirectDebitData = DirectDebitData(Some(List(DirectDebitDetails("Tester Surname", "404784", "70872490"))))
 
+  private val items: Seq[Item] = Seq(Item(Some(LocalDate.parse("2018-03-31"))))
+
   val financialData: FinancialData =
-    FinancialData("VRN", vrn.value, "VATC", "2019-08-20T10:44:05Z", Seq(Transaction("VAT Return Credit Charge", Option("18AC"))))
+    FinancialData("VRN", vrn.value, "VATC", "2019-08-20T10:44:05Z", Seq(Transaction("VAT Return Credit Charge", Option("18AC"), Option(items))))
```
Comment (AndyHWChung) [inline test/support/DesData.scala:44]: will the new items appear in all responses or is there a way to control whether it's there or not for testing purposes?

Comment (DreamcastKnight) [reply test/support/DesData.scala:44]: far as I can tell they'll always be there now, not sure if we can but that would be helpful!
I think even if we have an empty one in there from the des response we get an empty items object

Comment (AndyHWChung) [reply test/support/DesData.scala:44]: ah wait, hold on - for some reason I thought this was the stubs and this was the data being returned by the stub. Is there a way to control the clearing date in the stub for testing purposes?

**Review by AndyHWChung — APPROVED** [review]

**Pull request merged**